mesonbuild/dependencies/dev.py

```diff
@@ -32,30 +32,44 @@ def __init__(self, environment, kwargs):
         super().__init__('gtest', environment, 'cpp', kwargs)
         self.main = kwargs.get('main', False)
         self.src_dirs = ['/usr/src/gtest/src', '/usr/src/googletest/googletest/src']
-        self.detect()
+        self.detect(environment, kwargs)
```

Comment:
Unindented change of file mode to 0755?

Reply:
Yip, thanks, due to copying around files